cli/src/generate/templates/js-binding.cc

```diff
@@ -4,9 +4,20 @@ typedef struct TSLanguage TSLanguage;
 
 extern "C" TSLanguage *tree_sitter_PARSER_NAME();
 
+// tstag() {
+//   b2sum -l64 <(printf tree-sitter) <(printf "$1") | \
+//   awk '{printf "0x" toupper($1) (NR == 1 ? ", " : "\n")}'
+// }
+// tstag language # => 0x8AF2E5212AD58ABF, 0xD5006CAD83ABBA16
+const napi_type_tag LANGUAGE_TYPE_TAG = {
```

Comment:
A simple comment is enough here.
```suggestion
// "tree-sitter", "language" hashed with BLAKE2
const napi_type_tag LANGUAGE_TYPE_TAG = {
```